### DISCUSSION TITLE
Recommendations on switching author or account

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#11679] [Request Help]
My sense is that there's a considerable amount of manual action involved in having PRs and Dependency Dashboards deduped or adopted by the new bot account and/or git author when those change. Are there any tips or guidelines on how to go about with such a change (in Gitlab in my case)?

**Comment by rarkins** [comment]
It would be great if you can document your experimentation here so we can maybe add a documentation page on it.

First of all the author who commits vs the account which authenticates with the platform can be independent. You can use `gitIgnoredAuthors` to list your "old" git author if you want to change it.

You will also want to use `ignorePrAuthor` so that Renovate can "see" the old/existing PRs.

**Reply by astellingwerf** [reply]
K, thanks Rhys. Those properties should be of great help in my migration.

I'll try to keep a log of things I come across, and share them with you.

**Reply by astellingwerf** [reply]
Was it ever considered to add a `ignoreIssueAuthor` property (or the like) for the dependency dashboard to be adopted? (I know that Gitlab appears to disallow changing an issues author, so it'd mean you're stuck with the property.)

**Reply by viceice** [reply]
No, it makes no sense in most cases, as the new author can't change the old issue?

**Reply by astellingwerf** [reply]
Ah, I didn't realize that privilege lies solely with the author..

**Reply by viceice** [reply]
I think it depends on how much rights you give renovate user.

**Comment by astellingwerf** [comment]
I set the properties that @rarkins mentioned in my Gitlab environment. It then fails when reacting a PR for a branch for which there already is a PR (see [11679-PR-conlict.jsonlines.txt](https://github.com/renovatebot/renovate/files/7154076/11679-PR-conlict.jsonlines.txt)). Is there anything I can do about that in Renovate, or will I have to delete all branches and/or PRs to let the new account kick in?

**Reply by rarkins** [reply]
It seems that the findPr() function isn't finding the existing MR, but it should: https://github.com/renovatebot/renovate/blob/7801ae7c1652426de10557555fc7c71dbfd47f71/lib/platform/gitlab/index.ts#L440-L442

Unless GitLab defaults to that mode?

Where/how are you configuring `ignorePrAuthor`?

**Reply by astellingwerf** [reply]
I must admit, I initially forgot that property, but with it (set in a shared preset, and logged with the correct value in the config listing), the problem remains. See [11679-PR-conlict-console.jsonlines.txt](https://github.com/renovatebot/renovate/files/7156626/11679-PR-conlict-console.jsonlines.txt)

**Reply by rarkins** [reply]
Sorry but that log is too long for me to have time to look through

**Reply by astellingwerf** [reply]
I thought it best to include the entire repos debug log, so that all information you need would be in there. My issue is with the branch `renovate/artifactory.otxlab.net-docker-hardened-node-14.17.x`, of which I know that there have several previous incarnations of, but all under the previous bot account.

**Reply by astellingwerf** [reply]
It seems that upgrading to version 27.6.2 (from 25.56.2) may have resolved the issue.
